src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -43,26 +49,35 @@ export const AdminVenueView: React.FC = () => {
   } = useConnectCurrentVenueNG(venueId);
 
   const renderAdminVenueTabs = useMemo(() => {
-    return Object.entries(adminVenueTabLabelMap).map(([key, text]) => (
-      <Nav.Link
+    return Object.entries(adminVenueTabLabelMap).map(([key, label]) => (
+      <Link
         key={key}
-        className={classNames("AdminVenueView__tab", {
+        to={adminNGVenueUrl(venueId, key)}
+        className={classNames({
+          AdminVenueView__tab: true,
           "AdminVenueView__tab--selected": selectedTab === key,
         })}
-        eventKey={key}
       >
-        {text}
-      </Nav.Link>
+        {label}
+      </Link>
     ));
-  }, [selectedTab]);
+  }, [selectedTab, venueId]);
 
-  const selectTab = useCallback((tab: string) => {
-    setSelectedTab(tab as AdminVenueTab);
-  }, []);
+  const gotoHome = useCallback(() => history.push(adminNGRootUrl()), [history]);
 
-  const selectTiming = useCallback(
-    () => setSelectedTab(AdminVenueTab.timing),
-    []
+  const gotoSpaces = useCallback(
```

Comment:
`goToSpaces` same for the rest of the functions. You can turn it up a notch and change it to `navigateToSpaces` that way the name indicates history change.

Reply:
renamed @ a1bdfd1e5370a2260e69737437e6fe83d99dbf16. 

also `navigateToRun` 😄